roles/pulp_webserver/tasks/firewalld.yml

```diff
@@ -19,19 +19,22 @@
         state: started
         enabled: true
 
+    # not using immediate: true as causes error https://bugzilla.redhat.com/show_bug.cgi?id=1817205
     - name: Accept HTTP connections on port {{ pulp_webserver_http_port }}
       firewalld:
         port: '{{ pulp_webserver_http_port }}/tcp'
         permanent: true
-        immediate: true
         state: enabled
+        zone: public
```

Comment:
Umm, we basically have to use the user's current zone. What if their server is set to zone "FedoraServer" rather than "public".

Maybe this is messing up idempotency on debian-10?

Reply:
looks like that, I've used 'public' as it is upstream default, but this is neede anymore